src/parser.rs

```diff
@@ -40,7 +40,7 @@ pub fn initialize() -> Command {
             Arg::new("url")
                 .short('u')
                 .long("url")
-                .required_unless_present_any(["stdin", "resume_from"])
+                .required_unless_present_any(["stdin", "resume_from", "update_app"])
```

Comment:
can remove this, since we're using `.exclusive(true)` below

Reply:
When i'm removing this and making update_app exclucive(true), there is error

```
❯ ./feroxbuster --update
error: the following required arguments were not provided:
  --url <URL>

Usage: feroxbuster --update --url <URL>

For more information, try '--help'.
```

Reply:
If i'm not removing update_app from required_unless_present_any, the feature is working as expected

```
❯ ./feroxbuster -U
Checking target-arch... x86_64-macos
Checking current version... v2.9.1
Checking latest released version... v2.9.0

Feroxbuster 2.9.1 is up to date
```